content/concepts/xrpl-sidechains/cross-chain-bridges.md

```diff
@@ -0,0 +1,45 @@
+---
+html: cross-chain-bridges.html
+parent: xrpl-sidechains.html
+blurb: Cross-chain bridges for the XRP Ledger enable value in the form of XRP and other tokens (IOUs) to move efficiently between blockchains.
```

Comment:
Usage guidelines say we're trying to get away from the term IOU (though I personally like it). Safe to remove it here.